config/payments.php

```diff
@@ -2,4 +2,29 @@
 
 return [
 
+    /**
+     * User model
+     */
+    'user_model'     => \App\Models\User::class,
```

Comment:
Require update accordingly to https://www.notion.so/tger/Rules-c1a7ceb5485048d0b2b68ed4247f887b

Reply:
Done!